### DISCUSSION TITLE
Ability to build roads when "plaza" selected

### DISCUSSION THREAD

**Discussion by ipanic** [Keriew/augustus#259] [Ideas]
It is annoyed to build plaza roads with two actions. It would be better to have an ability to draw plaza directly on land. Price might be calculated as plaza + road on clear land

**Comment by brainwad** [comment]
It would break the ability to drag plazas as areas to plaza-up whole areas at once.

**Comment by crudelios** [comment]
Yeah, I'm not too keen on this feature either.

If anything I think we should restrict plaza creation like in Pharaoh, where you can only build them if roads are upgraded.

**Reply by ruideyllot** [reply]
Restricting, or calibrating, the desirability levels for plaza creation seems to be a good idea.
It always "triggered" me that, under the right conditions, one can go directly from dirt path to plaza.
The minimum desirability value for a plaza should be high enough to not allow us to "bypass" the upgraded road.
